src/py/flwr/server/superlink/state/state_test.py

```diff
@@ -409,6 +413,16 @@ def test_num_task_res(self) -> None:
         # Assert
         assert num == 2
 
+    def test_client_public_keys(self) -> None:
+        """Test client public keys store and get from state."""
+        state: State = self.state_factory()
+        key_pairs = [generate_key_pairs() for _ in range(3)]
+        public_keys = {public_key_to_bytes(pair[1]) for pair in key_pairs}
+
+        state.store_client_public_keys(public_keys)
+
+        assert state.get_client_public_keys() == public_keys
```

Comment:
Let's use the same Prepare / Execute / Assert comments here that we use in many other tests

Reply:
Done